.eslintrc

```diff
@@ -2,7 +2,9 @@
   "extends": "shellscape",
   "globals": {
     "document": true,
+    "ʎɐɹɔosǝʌɹǝs": true,
```

Comment:
interesting
whats is this about?

Reply:
https://github.com/shellscape/webpack-plugin-serve/blob/db6f09c3412cf77a7384feafd9c24be9a19e7a34/lib/index.js#L161 😉